.travis.yml

```diff
@@ -10,13 +14,16 @@ mysql:
   encoding: utf8
 
 cache:
+  yarn: true
```

Comment:
According to https://blog.travis-ci.com/2016-11-21-travis-ci-now-supports-yarn It looks like we should also add the node_modules directories.

Reply:
Yes maybe we should, even if the only package we're installing with npm is `yarn`. I'll add the line.

Reply:
Where are the packages stored that are installed by the individual calls to `yarn install`?

Reply:
yarn caches the packages in `$/.cache/yarn` which is covered by the travis shortcut `yarn: true`. when they don’t have to be downloaded any more, `yarn install` just copies them over to the local `node_modules` directories which saves quite some time. We can only guarantee proper builds if we throw away the local `node_modules` directories.

Reply:
> We can only guarantee proper builds if we throw away the local `node_modules` directories.

If that’s the case then yarn is broken.

Reply:
The problem is that downloading the dependencies is only about 50% of the time that’s spent on building the node_modules folder. (Just tried it for the message app on my computer.)